marlowe-dashboard-server/src/Marlowe/Run/Webserver/Wallet/CentralizedTestnet/Server.hs

```diff
@@ -0,0 +1,144 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE OverloadedStrings     #-}
```

Comment:
If you remove `{-# LANGUAGE OverloadedStrings #-}`, then you can also remove the several `:: Text` annotations. The protolude's `show` and `putStrLn` couldn't handle the ambiguity in string overloading, so if you make the strings into actual `String` types, then the ambiguity is removed.

Reply:
Nice catch! it worked fine :). I'll add it as part of the next PR so we don't have to re-approve this one